packages/generator/src/parser.ts

```diff
@@ -97,6 +98,10 @@ export function processApiFamily(
   const promises = [];
   const ramlFileFromFamily = apiFamilyConfig[apiFamily];
   _.map(ramlFileFromFamily, (apiMeta: RestApi) => {
+    if(!apiMeta.fatRaml.createdDate) {
```

Comment:
Why are we only checking for createdDate?

Reply:
createdDate is one of the fields that we reset when we are unable to get an asset from exchange. Since it is one of the mandatory fields, we shouldn't go forward if this is not set.